includes/class-sensei-course.php

```diff
@@ -2784,22 +2785,47 @@ public static function course_archive_featured_filter( $query ) {
 	}
 
 	/**
-	 * if the course order drop down is changed
+	 * Set the sorting options based on the query parameter and configuration in the Courses archive page.
 	 *
-	 * Hooked into pre_get_posts
+	 * Hooked into pre_get_posts.
 	 *
 	 * @since 1.9.0
 	 * @param WP_Query $query
 	 * @return WP_Query $query
 	 */
-	public static function course_archive_order_by_title( $query ) {
```

Comment:
So we can't really delete or rename this function since it's public and third party devs could be using it. We need to [deprecate](https://senseilms.com/documentation/deprecation-policy/) it instead.

Reply:
😮  right. This will byte me a lot...Thanks for the heads up!

Reply:
Also, since we deprecated this method, we should call the `_deprecated_function` function here.
It's to show a warning when using the method. ;)

Reply:
We could also add the `Deprecation` label to the PR, and add it in the PR description as the template.